cmd/test.go

```diff
@@ -236,7 +236,7 @@ func opaTest(args []string) int {
 			}
 		}
 	} else {
-		reporter = tester.JSONCoverageReporter{
+		reporter = tester.PrettyCoverageReporter{
```

Comment:
We will need to continue supporting the JSON output format when specified (eg: `--format json`). This one should be shown when the format is configured to be `pretty`.

This would introduce an (arguably) breaking change in the CLI behavior... @tsandall thoughts on keeping it default to `json` if maybe the `--format` flag was not set?

Reply:
Could you check the change I have made? I've kept `pretty` as the default because the help says so and I'm not really sure if this is the only instance where the default is set to `pretty`.

Reply:
I think its probably OK. We should however add a note to the CHANGELOG noting the new behavior.